packages/paste-website/src/pages/components/multiselect-combobox/index.mdx

```diff
@@ -124,8 +124,9 @@ Use a (Multiselect) [Combobox](/components/combobox) when:
 
```

Comment:
todo: Change "Use a (Multiselect) [Combobox](/components/combobox) when:" to:

`Use a Multiselect Combobox when:`